LINUXCOMPILE.md

```diff
@@ -3,7 +3,7 @@
 ### GNU Compiler
 ```
     # Ubuntu / Debian
-    sudo apt-get install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
+    sudo apt install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
```

Comment:
"apt" is preferred over "apt-get" for new users. 
Both are doing the same, but "apt" contains features from few different commands like apt-cache+apt-get (https://www.maketecheasier.com/apt-vs-apt-get-ubuntu/)